### DISCUSSION TITLE
Mails cannot send by Vaultwarden

### DISCUSSION THREAD

**Discussion by VaultSchulze** [dani-garcia/vaultwarden#5020] [Q&A]
Hi,
i am new at Vaultwarden and Linux, maybe you can help me or give some hints. I've set up Vaultwarden via Ubuntu 24.04 bash in our organisation. This should be available only internal. We are running a Windows Domain network. I am using Docker and Caddy for VW. Mails are hosted at All-Inkl and connected with Exchange Server 2016.

It is possible for me to login to the Vaultwarden Admin page by Browser. DNS is working via Windows DNS Server. But i cannot send a testmail or add any users by mail. For "Send test email" I got error message: Error sending SMTP test email 400 -     Unknown error
Here are my config files. Just let me know if you need some more information.

Ubuntu is setup via bash. I've already configured ufw:
![VW_ufw](https://github.com/user-attachments/assets/51511cb8-9608-4c6c-a93b-fa278d6c3ab3)

docker-compose.yml
![VW_DockerCompose](https://github.com/user-attachments/assets/b1a9ecd4-51f7-4311-8ded-747d757d4ae9)

Caddyfile
![VW_Caddyfile](https://github.com/user-attachments/assets/9097e27e-ba02-4cef-8240-52b545e92e41)

Could you please help me?

**Comment by stefan0xC** [comment]
Can you post the generated support string from the `/admin/diagnostics` page? That would maybe give us more information about what could be wrong. What do the Vaultwarden logs says when you send an test mail?
You can also enable `SMTP_DEBUG=true` to get more verbose infos if it's an SMTP error. https://github.com/dani-garcia/vaultwarden/blob/040e2a7bb0f2cc5012d46ca99283cf21fa06ed1a/.env.template#L540-L543
